README.rst

```diff
@@ -30,9 +30,70 @@ Run::
 
     python manage.py migrate djangocms_moderation
 
+REPLACE
 to perform the application's database migrations.
 
+Configuration
+============
+
+The following settings can be added to your project's settings file to configure django CMS Moderation's behavior:
+
+.. list-table::
```

Comment:
@sourcery-ai Can you add the defaults and the selection of available compliance backends?